### DISCUSSION TITLE
Unable to install the smartcontractkit/chainlink to my vs code

### DISCUSSION THREAD

**Discussion by QsAdedipe** [Cyfrin/foundry-full-course-cu#1601] [Q&A] [answered]
I am trying to install the smartcontractkit/chainlink-brownie to my vs code for the fundme exercise, below is the error that I am getting

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/97874921/196e5ad7-aae0-4f4d-adc5-51fe23c207fd)

**Comment by EngrPips** [comment, marked as answer]
Hello @QsAdedipe , this is a network issue. Please retry to install the package when your internet connection is stable

**Comment by QsAdedipe** [comment]
I've re-tried it but still having the same issue

**Reply by EngrPips** [reply]
Hello @QsAdedipe , Honestly it still the same reason `your network isn't stable` , whenever you want to install openzeppelin contracts you always need a very stable network or else it would always fail. I always face this issue too so much that i always wake at the middle of the night when everybody is asleep and the network is less congested so that i could install openzeppelin contract

**Comment by QsAdedipe** [comment]
Thank you, I will keep trying

On Mon, Apr 15, 2024 at 2:45 PM EngrPips ***@***.***> wrote:

> Hello @QsAdedipe <https://github.com/QsAdedipe> , Honestly it still the
> same reason your network isn't stable , whenever you want to install
> openzeppelin contracts you always need a very stable network or else it
> would always fail. I always face this issue too so much that i always wake
> at the middle of the night when everybody is asleep and the network is less
> congested so that i could install openzeppelin contract
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Cyfrin/foundry-full-course-f23/discussions/1601#discussioncomment-9118783>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXKXH2MLZSCT2B5LJXNHVLTY5PKX5AVCNFSM6AAAAABGGH64JSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCMJYG44DG>
> .
> You are receiving this because you were mentioned.Message ID:
> <Cyfrin/foundry-full-course-f23/repo-discussions/1601/comments/9118783@
> github.com>
>


-- 
QS Adedipe Oluwasina, MNIQS, RQS

**Reply by EngrPips** [reply]
Yeah, make sure to try whenever you think the internet connection should be stable

**Comment by QsAdedipe** [comment]
Thank you @EngrPips . It has be resolved

**Reply by EngrPips** [reply]
I am thrilled that you got it cracked. Keep crushing it, friend.

**Comment by QsAdedipe** [comment]
Haha, thanks!
